index.html

```diff
@@ -4297,6 +4299,46 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header">1.2.4</b> &mdash; <small><i>Jan. 9, 2016</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.2.3...1.2.4">Diff</a>
+    &mdash; <a href="https://cdn.rawgit.com/jashkenas/backbone/1.2.4/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        Fixed <tt>_removeModels</tt> regression which could cause an infinite loop
+        under certain conditions.
+      </li>
+      <li>
+        Added <tt>findIndex</tt> and <tt>findLastIndex</tt> Underscore methods to
+        <tt>Collection</tt>.
+      </li>
+      <li>
+        Fixed <tt>Model#id</tt> regression where <tt>id</tt> would be
+        unintentionally `undefined`.
+      </li>
+      <li>
+        Ensured <tt>Collection#_removeModels</tt> always returns an array.
+      </li>
+      <li>
+        Fixed <tt>Collection#_onModelEvent</tt> regression where triggering a
+        <tt>change</tt> event without a <tt>model</tt> would error.
+      </li>
+      <li>
+        Ensured <tt>Collection#reduce</tt> and <tt>Collection#reduceRight</tt>
+        work without an initial <tt>accumulator</tt> value.
+      </li>
+      <li>
+        Fixed <tt>Collection#set</tt> regression when <tt>parse</tt> returns a
+        falsy value.
+      </li>
+      <li>
+        Removed <tt>component</tt> package support.
+      </li>
+      <li>
+        Added <tt>previousModels</tt> and  <tt>addedModels</tt> attributes to <tt>Collection</tt> update events.
```

Comment:
It's `options.changes` which includes an arrays ala `{added: [...], removed: [...], merged: [...]}`

So his should be something like: added `options.changes` to `Collection` "update" event which include added, removed, and merged models.

A similar change should be made here: https://github.com/jdalton/backbone/blob/changelog/index.html#L1005